### PR TITLE
Modifies email templates

### PR DESCRIPTION
**Pull request by GrantComm**
Changed the email template for emails that are sent when an interview is scheduled to include the gvc link

### CODE REVIEW

File: src/main/resources/templates/email/NewInterview_Interviewee.txt [hunk 1]
```diff
@@ -15,4 +15,6 @@ After interviewing:
  - Your interviewer will write up a feedback report. Expect to see this in your inbox shortly.
  - Send the interviewer feedback through this form: {{form_link}}.
 
-A Google Calendar event titled "InterviewMe Interview {{interviewer_first_name}} / {{interviewee_first_name}}" was added to your Google Calendar. Please reach out to interviewme.business@gmail.com if you have any questions.
+Your interview will be conducted in this Google Virtual Chat: {{chat_link}}.
+
+Please reach to interviewme.business@gmail.com if you have any questions.
```
Comment (meagancheese) [inline src/main/resources/templates/email/NewInterview_Interviewee.txt]: Nit: missing "out" as in "reach out"

File: src/main/resources/templates/email/NewInterview_Interviewer.txt [hunk 2]
```diff
@@ -15,4 +15,6 @@ After interviewing:
  - Your interviewee will write up a feedback report. Expect to see this in your inbox shortly.
  - Send the interviewee feedback through this form: {{form_link}}.
 
-A Google Calendar event titled "InterviewMe Interview {{interviewer_first_name}} / {{interviewee_first_name}}" was added to your Google Calendar. Please reach out to interviewme.business@gmail.com if you have any questions.
+Your interview will be conducted in this Google Virtual Chat: {{chat_link}}.
+
+Please reach to interviewme.business@gmail.com if you have any questions.
```
Comment (meagancheese) [inline src/main/resources/templates/email/NewInterview_Interviewer.txt]: Same here.

**Review by meagancheese — APPROVED** [review]
Small fix, otherwise LGTM.

File: src/main/resources/templates/email/NewInterview_Interviewer.txt (hunk 2 above)
Comment (GrantComm) [reply src/main/resources/templates/email/NewInterview_Interviewer.txt]: Done

File: src/main/resources/templates/email/NewInterview_Interviewee.txt (hunk 1 above)
Comment (GrantComm) [reply src/main/resources/templates/email/NewInterview_Interviewee.txt]: Done

**Review by apluscs — APPROVED** [review]

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**